### PR TITLE
EUI-2766: Task Manager

### PR DESCRIPTION
**Pull request by paul-graham**
### JIRA link ###
https://tools.hmcts.net/jira/browse/EUI-2766

**Also:**
- https://tools.hmcts.net/jira/browse/EUI-2764
- https://tools.hmcts.net/jira/browse/EUI-2889
- https://tools.hmcts.net/jira/browse/EUI-2961

### Change description ###
The Task Manager component with filtering by location and caseworker.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]

File: src/work-allocation/services/session-storage.service.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
```
Comment (udaydenduluri33) [inline src/work-allocation/services/session-storage.service.ts:1]: Should this be moved to common-library at some point?

**Review by udaydenduluri33 — APPROVED** [review]
LGTMp

File: src/work-allocation/services/session-storage.service.ts (hunk 1 above)
Comment (paul-graham) [reply src/work-allocation/services/session-storage.service.ts:1]: If it's going to be used elsewhere in MC, it should be moved to a shared area within MC. If it's going to be used in one of the other applications, the common library makes sense.

For now, though, this is the only use of session storage I've found, which is why it's sitting where it is. :)

**Review by udaydenduluri33 — COMMENTED** [review]
Flawless

**Pull request merged**